packages/notifi-react-card/lib/components/NotifiTelegramInput.tsx

```diff
@@ -25,14 +27,29 @@ export const NotifiTelegramInput: React.FC<NotifiTelegramInputProps> = ({
   disabled,
   intercomTelegramInputStyle,
   intercomTelegramInputContainerStyle,
+  intercomView,
 }: NotifiTelegramInputProps) => {
-  const { telegramId, setTelegramId } = useNotifiSubscriptionContext();
+  const { intercomCardView, telegramId, setTelegramId } =
+    useNotifiSubscriptionContext();
 
   return (
     <>
-      <label className={clsx('NotifiSmsInput__label', classNames?.label)}>
-        {copy?.label}
-      </label>
+      {intercomView ? (
+        intercomCardView.state === 'settingView' ? (
+          <div
```

Comment:
Similar here as well.

Nit: This Resend verification link will appear if in the data, the destination has not been verified. You can take steps to start implementing the logic by passing in a isVerified data and/or setting a setState.